src/buttonMaker/IconForm.jsx

```diff
@@ -0,0 +1,98 @@
+import * as allIcons from "@chakra-ui/icons";
+import "./buttonMaker.css";
+import { changeForm, clickForm } from "./formLogic";
+
+// A module of all the availabe icons
+let { Icon, createIcon, ...iconsModule } = allIcons;
+
+// A list of all the icon names
+let iconsName = Object.keys(iconsModule).map((e) => e);
+
+// A dropdown for the user to pick their icon
+function DropdownItem({ iconName, Icon, formDispatch }) {
+    return (
+        <button
+            name={"icon"}
+            id={iconName.replace(/\s/g, "")}
+            onClick={(e) => clickForm(e, formDispatch)}
+        >
+            <Icon /> &nbsp;
```

Comment:
suggestion: rather than putting a space with this trick, put some margin on the icon!